Makefile

```diff
@@ -1,36 +1,59 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         ::::::::             #
+#    Makefile                                           :+:    :+:             #
+#                                                      +:+                     #
+#    By: abeznik <abeznik@student.codam.nl>           +#+                      #
+#                                                    +#+                       #
+#    Created: 2023/02/14 10:25:24 by abeznik       #+#    #+#                  #
+#    Updated: 2023/02/14 10:43:47 by abeznik       ########   odam.nl          #
+#                                                                              #
+# **************************************************************************** #
+
 NAME		:= cub3D
-CFLAGS		:= -Wextra -Wall -Werror
+
+RED 		:=	\033[1;31m
+GRN 		:=	\033[1;32m
+BLUE		:=	\033[1;36m
+YEL 		:=	\033[0;33m
+DEF 		:=	\033[0m
+
+CFLAGS		:= -Wextra -Wall -Wunreachable-code -Ofast -g3 #-Werror
 CFLAGS		+= $(if $(FSAN) , -fsanitize=address -g)
 CFLAGS		+= $(if $(DEBUG) , -g)
-LIBMLX		:= ./lib/MLX42
-LIBFT		:= ./lib/libft
-MLXFLAGS:= -lglfw3 -framework Cocoa -framework OpenGL -framework IOKit -g
 
-HEADERS	:= $(addprefix -I , \
-			  libft \
-			  MLX42/include/MLX42)
-LIBS	:= $(LIBFT)/libft.a $(LIBMLX)/libmlx42.a
-SRCS	:= $(shell find ./srcs -iname "*.c")
-OBJS	:= ${SRCS:.c=.o}
+LIBMLX		:= ./libs/MLX42
+LIBFT		:= ./libs/libft
+LIBS		:= $(LIBFT)/libft.a $(LIBMLX)/build/libmlx42.a -ldl -lglfw3 -pthread -lm
```

Comment:
why do we need pthread and language model?

Reply:
these are actually the flags for linux, ill change these back to the MacOS' ones